src/platforms/flutter/migration.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Migration Guide
+sidebar_order: 1000
+redirect_from:
+  - /platforms/flutter/migrate/
+---
+
+## Migrating from `sentry_dart` `4.0.x` to `sentry_dart` `5.0.0`
+
+In addition to the changes of [sentry_dart](/platforms/dart/migrate/)
+there are the following changes:
+
+- `SentryFlutterOptions.enableLifecycleBreadcrumbs` was removed. Use `SentryFlutterOptions.enableAppLifecycleBreadcrumbs` instead.
+- The Web Plugin Registrant import changed from 'package:sentry_flutter/src/sentry_flutter_web.dart'; to 'package:sentry_flutter/sentry_flutter_web.dart';
+  - This could lead to breaking changes. Typically it shouldn't because the referencing file is auto-generated.
```

Comment:
```suggestion
  - This change may lead to breaking changes. In most cases, however, this change won't lead to breaking changes since the referencing file is auto-generated.
```